### ISSUE TITLE
"flip id" should just be "flip"

### ISSUE THREAD

**Issue by stuartpb** [bmillwood/pointfree#25]
Passing `flip' x y = y x` on pointfree.io returns `flip' = flip id` instead of just `flip' = flip`, which would be equivalent, and neater.

**Comment by camilstaps** [comment]
`flip' x y = y x` is not equivalent to `flip f x y = f y x`. Note that `id x = x` and `id' x y = x y` *are* equivalent, which explains that `flip id' x y = id' y x = y x = flip' x y`.

**Comment by stuartpb** [comment]
Right, I'd forgotten about the `f` that `flip` takes.

Is there no name for `flip id`, then? I'd heard some rumbling about `&`, but I can't find any documentation on it at the moment.

**Comment by camilstaps** [comment]
Yes, you could say that `flip' = &`, but that operator is only available in the newest GHC, so not sure if the maintainer wants to use it in pointfree already. You can find it on the beta version of hoogle: http://hoogle.haskell.org/?hoogle=%26&scope=set%3Astackage

**Comment by bmillwood** [comment]
I think it's fine to use `&` at this stage, but I don't think I'll get round to spending time on it personally :)
